src/main/resources/application.properties.example

```diff
@@ -1 +1,9 @@
+moderation.warningperiod=1209600
```

Comment:
(me being nitpicky :P ) camelcase or other-case :P

Reply:
everything in spring must be lowercase, they follow an explicitly naming convention. In docker you can then configure environment variables by making everything uppercase and replacing the dots with underscores